internal/kafka/internal/clusters/cluster_builder.go

```diff
@@ -87,7 +87,7 @@ func (r clusterBuilder) NewOCMClusterFromCluster(clusterRequest *types.ClusterRe
 	// Set compute node size
 	clusterBuilder.Nodes(clustersmgmtv1.NewClusterNodes().
 		ComputeMachineType(clustersmgmtv1.NewMachineType().ID(computeMachineType)).
-		AutoscaleCompute(clustersmgmtv1.NewMachinePoolAutoscaling().MinReplicas(6).MaxReplicas(18)))
+		AutoscaleCompute(clustersmgmtv1.NewMachinePoolAutoscaling().MinReplicas(3).MaxReplicas(18)))
```

Comment:
Can we define this as a constant? Will this be the same configuration for gcp too?

Reply:
> Can we define this as a constant? 

Good idea. I'll do so.

> Will this be the same configuration for gcp too?

Good question. There hasn't been any indication to change this (especially the maximum) for gcp.

Reply:
raised the configuration question for gcp internally.